File: Card.java
public class Card {

    private String question;
    private boolean answeredCorrectlyLastRound;

    public Card(String question, boolean answeredCorrectlyLastRound) {
        this.question = question;
        this.answeredCorrectlyLastRound = answeredCorrectlyLastRound;
    }

    public boolean wasAnsweredCorrectlyLastRound() {
        return answeredCorrectlyLastRound;
    }

    public String getQuestion() {
        return question;
    }
}

File: Card.ts
export class Card {
  question: string;
  answeredCorrectlyLastRound: boolean;

  constructor(question: string, answeredCorrectlyLastRound: boolean) {
    this.question = question;
    this.answeredCorrectlyLastRound = answeredCorrectlyLastRound;
  }

  wasAnsweredCorrectlyLastRound(): boolean {
    return this.answeredCorrectlyLastRound;
  }
}